ai/default/aisettler.cpp

```diff
@@ -377,7 +377,7 @@ static struct cityresult *cityresult_fill(struct ai_type *ait,
     }
 
     if (result->tdc_hash->contains(cindex)) {
-      NFCPP_FREE(result->tdc_hash->value(cindex));
```

Comment:
Why were these macros removed ? They do the exact same thing as the delete.

Reply:
Readability

Reply:
So, The aim is to remove all the macros ? Someone should have told me that when I was fixing the malloc-dealloc mismatches lol

Reply:
Over time yes. I don't even understand what the various prefixes are for, need to look at the definition every time :D